.github/workflows/ci-pr-only-nns-team-reminder.yml

```diff
@@ -47,24 +47,47 @@ jobs:
             console.log("Adding reminder to update unreleased_changelog.md...");
             const reminderText = `
               If this pull request changes the behavior of any canister owned by
-              the Governance team in an externally visible way, remember to
-              update the corresponding unreleased_changelog.md file(s).
+              the Governance team in an externally visible way, make sure that
+              you have done the following:
+
+              1. Add entry(s) to corresponding `unreleased_changelog.md`
+                 file(s). This is only necessary if the code changes result in
+                 externally visible behavior changes.
+
+              2. If there are behavior changes, are they breaking? If so, are
+                 your clients ready for them? If not, put your changes behind a
+                 "feature flag". If you have a feature flag, then, do NOT add
+                 entries to `unreleased_changelog.md` in this PR, but rather, in
+                 the later PR, when you set the flag to "enable".
+
+              3. Is data migration needed? (If so, make sure it is part of this
+                 PR.)
+
+              4. Does this require security review? At the very least, you can
+                 tell security team about this PR, and let them decide whether
+                 the risk warrants their review.
 
               To acknowldge this reminder (and unblock the PR), dismiss this
               code review by going to the bottom of the pull request page, look
               for where it says this bot is requesting changes, click the three
-              dots on the right, select "Dismiss review", and supply one of the
-              following reasons:
+              dots on the right, select "Dismiss review", and for each of the
+              numbered items listed above, supply one of the following reasons:
 
-              1. Done.
+              * Done.
 
-              2. No canister behavior changes.
+              * $REASON_WHY_NO_NEED. E.g. for `unreleased_changelog.md`, "No
+                canister behavior changes.", or for item 2, "Existing APIs
+                behave as before.".
 
               To be more precise, "externally visible behavior change" usually
               means that responses differ in some way. However, "externally
               visible behavior change" is not limited to that. For example, it
               could also means that the canister makes different requests to
               other canisters.
+
+              For a more comprehensive checklist, [see here][checklist].
+
+              [checklist]: https://docs.google.com/document/d/1YeeCzVDB3PmvCGvV0OqZpV-R5F0fMGahoTlmXaIjj_U/edit?tab=t.0#heading=h.thlqmyi5yk1
```

Comment:
Do we have a precedence of pointing to an internal doc?
I kind of like that we point to the "more informatoin", but was just wondering if this is weird in open source code (as the doc cannot be accessed by exteranl readers)

Reply:
Fair.

OTOH, we already link to Jira from PR descriptions, which is worse, because PR descriptions make it into git. Whereas, this checklist link only appears as a code review, which does not end up in git; it only lives in GitHub.